README.md

```diff
@@ -1,2 +1,5 @@
-# insta-hack
```

Comment:
A project has been created to hack Facebook by guessing and in the near future I will make changes to the project in order to hack Instagram through the work of collecting information in order to make a list of passwords according to the information and sources for each account